CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - The `Status` type was added to the `go.opentelemetry.io/otel/sdk/trace` package to represent the status of a span. (#1874)
 - The `SpanStub` type and its associated functions were added to the `go.opentelemetry.io/otel/sdk/trace/tracetest` package.
   This type can be used as a testing replacement for the `SpanSnapshot` that was removed from the `go.opentelemetry.io/otel/sdk/trace` package. (#1873)
+- An example of using OpenTelemetry go in "Passthrough" mode. (#1912)
```

Comment:
```suggestion
- An example of using OpenTelemetry Go as a trace context forwarder. (#1912)
```

This language is more aligned with the description in the [trace context spec](https://www.w3.org/TR/trace-context/#design-overview) (forwarding vs. participating).